chalicelib/lib/helpers.py

```diff
@@ -51,32 +52,43 @@ def parse_date(date: str, format_str: str = "%Y-%m-%d") -> dict:
     "today"
     "2019-01-01"
     "2019-01-01:2019-01-02"
+    :returns:
+             single -> list(date: datetime)
+             range -> list(date_from: datetime, date_to: datetime)
     """
 
-    dates = list()
+    dates = {"to": None, "from": None}
+
+    # handle aliases
     if date == "today":
-        dates.append(datetime.now())
+        dates['to'] = datetime.now()
+        dates['from'] = datetime.now()
```

Comment:
The function should return the dict here to prevent it from entering the else state in the if statement later

Reply:
Also You could write it as:
```
date = datetime.now()
return dict(to=date, from=date)
```

Reply:
Since "from" is a reserved keyword it can't be used as a argument key to the dict function

Reply:
`from_date` and `to_date` maybe?

Reply:
Or `return {'from_date: date, 'to_date': date }` since `from` isn't OK to use with `dict()`

Reply:
Totally missded that you had an if/elif. I would suggest to use return early in favor of this, but the choice is yours